src/back-end/config.ts

```diff
@@ -51,6 +51,12 @@ export const ENV = NODE_ENV;
 
 const logger = makeDomainLogger(consoleAdapter, "back-end:config", ENV);
 
+export const PROD_LOG_DEBUG =
+  parseBooleanEnvironmentVariable(get("PROD_LOG_DEBUG", "0")) ?? false;
```

Comment:
I think you could generalize this to `LOG_DEBUG` and have the default value be `ENV === "development" ? "1" : "0"` instead of `"0"`. Then, in `logger/index.ts` you wouldn't have to look at the `ENV` config var, you could only look at `LOG_DEBUG` to turn debug logging on/off.

Reply:
Good idea, I'll try that out.

Reply:
I've made the suggested changes re: logging formatting and config parameter mentioned above @dhruvio , thanks!